### ISSUE TITLE
check off checklist items

### ISSUE THREAD

**Issue by jack-michaud** [OfficerHalf/obsidian-trello#9]
Firstly, PHENOMENAL plugin. I'm using it via git clone, but this is definitely ready to be on the community plugin list even in its current state. It's feature complete enough to be extremely useful for my needs.

 Enhancement Request: I'd like to be able to check off checklist items from within the Trello sidebar. This would let me indicate progress on a card without opening the web interface.

**Comment by OfficerHalf** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [checklist support](https://trello.com/c/HS0UmMbB/10-checklist-support)

**Comment by OfficerHalf** [comment]
@jack-michaud would you want to be able to view more than one checklist at once on a single card? Or just one at a time? Trying to figure out how to manage this.

**Comment by jack-michaud** [comment]
@OfficerHalf I would have a need for multiple checklists if I wanted to have Acceptance Criteria and/or Subtasks. I don't think I'd need to view multiple at the same time, though.

I wonder if a dropdown selector for different checkboxes would be a good solution? 
![image](https://user-images.githubusercontent.com/13189762/135545792-b72eb2e1-7ccf-4a4b-aab0-44537892d4f3.png)

**Comment by OfficerHalf** [comment]
That could work. I could also see an accordion type thing:

```
> Checklist 1 (progress %)
+ Checklist 2
=======Progress % =========
[ ] Task 1
[ ] Task 2
[ ] Task 3
> Checklist 3 (progress %)
```

**Comment by jack-michaud** [comment]
Oh, an accordion would be cool (especially with a progress indicator when collapsed).

IMO, go for the list if it's easiest. If it gets too dense, it can always be updated later!

**Comment by OfficerHalf** [comment]
Closed by #27

**Comment by OfficerHalf** [comment]
Added in v1.4.0. Let me know if you have any issues. 🙂
